lib/src/utils/package_info.dart

```diff
@@ -0,0 +1,15 @@
+/// A utility class that provides static information about the package.
+///
+/// This class contains the name and version of the package, which
+/// can be used for logging, attaching to requests, or displaying in the UI.
+class PackageInfo {
+  /// The name of the package.
+  ///
+  /// This constant holds the package's name as defined in `pubspec.yaml`.
+  static const String packageName = 'omise_flutter';
```

Comment:
I believe the package name standard which we pass in user agent is `Omise<Platform/Language>`, isn't it?

Reply:
updated, please check.